SpatialGDK/Source/SpatialGDKServices/Private/SSpatialOutputLog.cpp

```diff
@@ -20,6 +20,7 @@ DEFINE_LOG_CATEGORY(LogSpatialOutputLog);
 static const FString LocalDeploymentLogsDir(FPaths::Combine(SpatialGDKServicesConstants::SpatialOSDirectory, TEXT("logs/localdeployment")));
 static const FString LaunchLogFilename(TEXT("launch.log"));
 static const float PollTimeInterval(0.05f);
+auto ErrorLogFlagInfo = TTuple<bool, FString>(false, "");
```

Comment:
We tend to not use `auto` in the GDK.
 ```suggestion
TTuple<bool, FString> ErrorLogFlagInfo;
```